core/templates/pages/community-dashboard-page/translation-language-selector/translation-language-selector.component.html

```diff
@@ -0,0 +1,135 @@
+      arrow_drop_down
+    </span>
+  </div>
+
+  <div *ngIf="dropdownShown" class="oppia-translation-language-selector-dropdown-container protractor-test-language-selector-dropdown">
+    <div *ngIf="featuredLanguages.length > 0" class="protractor-test-featured-language-container">
+      <div class="oppia-translation-language-selector-dropdown-label">Most needed</div>
+      <div *ngFor="let featuredOption of featuredLanguages; index as i">
+        <div class="oppia-translation-language-selector-dropdown-option protractor-test-featured-language"
+             *ngIf="languageIdToDescription[featuredOption.languageCode]"
+             (click)="selectOption(featuredOption.languageCode)"
+             [ngClass]="{'oppia-translation-language-selector-dropdown-option-selected' : featuredOption.languageCode === activeLanguageCode}">
+          <span class="material-icons oppia-translation-language-selector-info-icon protractor-test-featured-language-tooltip"
+                *ngIf="featuredOption.explanation"
+                (mouseenter)="showExplanationPopup(i)"
+                (mouseleave)="hideExplanationPopup()">
+            info
+          </span>
+          {{languageIdToDescription[featuredOption.languageCode]}}
+        </div>
+      </div>
+    </div>
+
+    <div class="oppia-translation-language-selector-dropdown-label">All languages</div>
+    <div class="oppia-translation-language-selector-dropdown-option protractor-test-language-selector-option"
+         *ngFor="let option of options"
+         (click)="selectOption(option.id)"
+         [ngClass]="{'oppia-translation-language-selector-dropdown-option-selected' : option.id === activeLanguageCode}">
+      {{option.description}}
+    </div>
+  </div>
+</div>
+
+<style>
+  .oppia-translation-language-selector-container {
+    background-color: #fff;
+    border-radius: 4px;
+    box-shadow: 0 0 4px rgba(0, 0, 0, 0.25);
+    display: flex;
+    justify-content: space-between;
+    padding: 10px 20px;
+    position: relative;
+  }
+
+  .oppia-translation-language-selector-inner-container {
+    display: flex;
+    flex: 1;
+    justify-content: space-between;
+  }
+
+  .oppia-translation-language-selector-inner-container-arrow {
+    color: #009688;
+  }
+
+  .oppia-translation-language-selector-dropdown-container {
+    background-color: #fff;
+    border-radius: 4px;
+    box-shadow: 0 0 4px rgba(0, 0, 0, 0.25);
+    height: 40vh;
+    left: 0;
+    overflow-y: scroll;
+    padding: 8px 12px;
+    position: absolute;
+    right: 0;
+    top: 100%;
+  }
+
+  .oppia-translation-language-selector-dropdown-container::-webkit-scrollbar {
+    width: 7px;
+  }
+  .oppia-translation-language-selector-dropdown-container::-webkit-scrollbar-track {
+    border-radius: 5px;
+    box-shadow: inset 0 0 3px grey;
+  }
+  .oppia-translation-language-selector-dropdown-container::-webkit-scrollbar-thumb {
+    background: grey;
```

Comment:
Use hex value (also instead of rgb(..., ..., ...) can we use hex for consistency?)

Reply:
Done